package.json

```diff
@@ -333,6 +339,7 @@
     "jsdom-global": "3.0.2",
     "json-loader": "0.5.7",
     "konami-js": "0.0.2",
+    "levenary": "^1.1.1",
```

Comment:
What's this for?

Reply:
Not too sure -- things blew up without it; had to do the same over in palette.

Reply:
lol 

```
% yarn why levenary
yarn why v1.21.1
[1/4] 🤔  Why do we have the module "levenary"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "levenary@1.1.1"
info Has been hoisted to "levenary"
info This module exists because it's specified in "devDependencies".
```